core/domain/exp_domain.py

```diff
@@ -2874,6 +2927,27 @@ def _convert_states_v13_dict_to_v14_dict(cls, states_dict):
                 }
         return states_dict
 
+    @classmethod
+    def _convert_states_v14_dict_to_v15_dict(cls, states_dict):
+        """Converts from version 14 to 15. Version 15 renames the "correct"
+        field in answer groups to "labelled_as_correct" and (for safety) resets
+        all "correct" values to False.
```

Comment:
...sets all "labeled_as_correct" values to False?

Reply:
Done, thanks!